README.md

```diff
@@ -244,19 +276,75 @@ struct GetUserRequest: Request {
+Sometimes, when decoding an array of objects, you may not want to fail the entire request if some of those objects fail to decode. For example, the following `json` would fail to decode using standard decoding because the second post is missing the content.
+
+```json
+{ 
+    posts: [
+        { 
+        "title": "Super cool cat."
+        "content": "Info about a super cool cat."
+        },
+        {
+        "title": "Even cooler cat."
+        }
+    ]
+}
+
+```
 
 To do this, you can set your Request's `arrayDecodeStrategy` to `.lossy` to return any elements that succeed to decode.
 
-Not that this will only work if your `RawResource: Sequence` or `RawResource: SmartUnwrap<Sequence>`. For better support of decoding nested, lossy, arrays we recommend checking out [Better Codable](https://github.com/marksands/BetterCodable)  
+```swift
+struct Post: {
+   let title: String
+   let content: String
+}
+
+struct GetPostsRequests: {
+typealias RawResource: SmartUnwrap<[Post]>
+typealias FinalResource: [Post]
+
+var arrayDecodingStrategy: ArrayDecodingStrategy: { return .lossy }
+}
+```
+
+Note that this will only work if your `RawResource` is `RawResource: Sequence` or `RawResource: SmartUnwrap<Sequence>`. For better support of decoding nested, lossy arrays we recommend checking out [Better Codable](https://github.com/marksands/BetterCodable). Also, at this time, Netable doesn't support partial decoding for GraphQL requests.
+
+#### Create a LossyArray directly within your object
+
+Using `.lossy` as our `arrayDecodingStrategy` works well for objects that are being decoded as an array. We've added support to allow for partial decoding of objects that _contain_ arrays.
+
+```swift
+struct User: Decodable {
+    let firstName: String
+    let lastName: String
+    let bio: String
+    let additionalInfo: LossyArray<AdditionalInfo>
+}
+
+struct UserLoginData: Decodable, Hashable {
+    let age: Int
+    let gender: String
+    let nickname: String
+}
+```
+
+Note: to access the LossyArray's elements, you have to access `.element` within, like so.
+
+```swift
+    ForEach(user.additionalInfo.element, id: \.self) {
+    // ..
+    }
+```
 
 ### Handling Errors
 
 In addition to handling errors locally that are thrown, or returned through `Result` objects, we provide two ways to handle errors globally. These can be useful for doing things like presenting errors in the UI for common error cases across multiple requests, or catching things like failed authentication requests to clear a stored user.
 
 #### Using `requestFailureDelegate`
 
-See [GlobalRequestFailureDelegate](https://github.com/steamclock/netable/blob/master/Netable/NetableExample/ViewController/RootTabBarController.swift) in the Example project for a more detailed example.
+See [GlobalRequestFailureDelegate](https://github.com/steamclock/netable/blob/aeo/86-new-example-project/Netable/Example/Services/ErrorService.swift) in the Example project for a more detailed example.
```

Comment:
You'd mentioned it in Slack, but I want to point out we'll need to switch these over to master before merging

Reply:
So I can swap these right to `master/netable` now? I guess as long as I don't change the files or directories, the paths won't change so the links won't break, correct? That's a dumb question I know 😅

Reply:
Yeah, might as well swap them over now